src/LanguageServer/ProtocolUnitTests/OnAutoInsert/OnAutoInsertTests.cs

```diff
@@ -35,8 +36,8 @@ void M()
             class A
             {
                 /// <summary>
-            /// $0
-            /// </summary>
+                /// $0
```

Comment:
consequence of including indentation now that we can tell vscode not to change our edits